docs/user_guide/1_getting_started_hydromt/index.rst

```diff
@@ -0,0 +1,15 @@
+Getting started with HydroMT
+================================
+
+The core functionalities of HydroMT, such as the command-line interface (CLI), data catalog management, and region definitions, are documented in the main HydroMT documentation.
+These topics are shared across all HydroMT model plugins (including HydroMT-Wflow) and are maintained in one central location to ensure consistency.
+Refer to the links below for the official and up-to-date information.
```

Comment:
The toctree is hidden so this sentence is a little strange. Maybe add a grid? See examples from user_guide/index